syntax = "proto3";

package request;

option go_package = "github.com/MartinSimango/goftqproto/internal/request/genrequest";

import "ftqproto/RequestFile.proto";

message CreateRequest {
    string sourceFilePath = 1;
    string destinationFilePath = 2;
    repeated RequestFile files = 3;
}